tests/manual/tablemocking.html

```diff
@@ -0,0 +1,56 @@
+<head>
+	<meta http-equiv="Content-Security-Policy" content="script-src 'self' https://ckeditor.com 'unsafe-inline' 'unsafe-eval'">
+</head>
+
+<style>
```

Comment:
```suggestion
<style>
```

If the above is not needed :point\_up: .